packages/components/src/primitives/svg/index.native.js

```diff
@@ -19,7 +19,7 @@ export {
 export const SVG = ( props ) => {
 	const colorScheme = props.colorScheme || 'light';
 	const stylesFromClasses = ( props.className || '' ).split( ' ' ).map( ( element ) => styles[ element ] ).filter( Boolean );
-	const defaultStyle = props.active ? styles[ 'is-active' ] : styles[ 'components-toolbar__control-' + colorScheme ];
+	const defaultStyle = props.__unstableActive ? styles[ 'is-active' ] : styles[ 'components-toolbar__control' ];
```

Comment:
This looks like it would break the toolbar button's Dark Mode scheme on iOS. https://github.com/WordPress/gutenberg/pull/17356

This last lint error looked suspicious: 
```
/src/primitives/svg/index.native.js
734[0]   20:8  error  'colorScheme' is assigned a value but never used  no-unused-vars
```

Reply:
Indeed it looks broken.
@marecar3 - Any particular reason of why was it removed?
This works locally:
```suggestion
	const defaultStyle = props.__unstableActive ? styles[ 'is-active' ] : styles[ 'components-toolbar__control-' + colorScheme ];
```

Reply:
Thanks @etoledom  for pointing on that one. 

Honestly, I am not 100% sure that @gziolo and @youknowriad 
want this `props.__unstableActive` to be merged into the `master` branch.

@gziolo is trying to find some proper solution but not sure if that will block us too long with `merging` process.

Reply:
>Indeed it looks broken.
@marecar3 - Any particular reason of why was it removed?
This works locally:

No particular reason, sorry about that!
Not sure why my commit removed that one as I didn't have intention to do that.

Reply:
@gziolo should we revert this change before merging?

Reply:
See https://github.com/WordPress/gutenberg/pull/17676#issuecomment-539380426